service_container/alias_private.rst

```diff
@@ -111,12 +111,16 @@ services.
     .. code-block:: php
 
         use AppBundle\Mail\PhpMailer;
+        use Symfony\Component\DependencyInjection\Alias;
         use Symfony\Component\DependencyInjection\Definition;
 
         $container->setDefinition('app.phpmailer', new Definition(PhpMailer::class));
 
         $containerBuilder->setAlias('app.mailer', 'app.phpmailer');
 
+        // private aliases are created passing 'false' to Alias() second argument
+        $containerBuilder->setAlias('app.mailer', new Alias('app.phpmailer', false));
+
 This means that when using the container directly, you can access the
```

Comment:
Given this explanation, I wonder if the above place is the right place to show this?